README.md

```diff
@@ -0,0 +1,24 @@
+# Bot Deployed By Nethermind Deployer Bot
+
+## Description
+
+This bot detects creations of forta bots by the forta deployer address.
+
+## Supported Chains
+
+- Polygon
+
+## Alerts
+
+- Forta-Bot-Deployed
+  - Fired when the function `createAgent` is called on the Forta contract
+  - Severity is always set to "Info"
+  - Type is always set to "Info"
+  - Metadata contains:
+    - Provides the deployer, agentId and chainIds
+
+## Test Data
+
+The agent behaviour can be verified with the following transactions:
```

Comment:
```suggestion
The bot behaviour can be verified with the following transactions:
```
a small one but we use the word 'bot' instead of 'agent' in Readme & descriptions